src/components/stack-chart/Canvas.js

```diff
@@ -210,6 +210,13 @@ class StackChartCanvasImpl extends React.PureComponent<Props> {
       TEXT_CSS_PIXELS_OFFSET_START * devicePixelRatio;
     const textDevicePixelsOffsetTop =
       TEXT_CSS_PIXELS_OFFSET_TOP * devicePixelRatio;
+    let categoryForUserTiming = categories.findIndex(
```

Comment:
This is a behavior change! I don't think anyone picked a category for these `UserTiming` changes, and it was defaulting to Idle. I chose JavaScript. It changes the color of the User Timing.

Reply:
Oh, weird. Thanks for fixing this!